managing_providers/_topics/embedded_workflows.md

```diff
@@ -280,6 +280,21 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
   }
 }
 ```
+#### Running an Embedded Workflow on the Openshift Container Platform (OCP) 
+
+If the user is running an embedded workflow on OCP, as opposed to appliances, and is using a docker repository requiring a custom pull secret for their container images, the following steps are necessary:
+
+1. create a docker pull secret for the repository in question in the same namespace as IM, for example: 
+
+    ```
+    oc create secret docker-registry <pull-secret-name> --docker-server=<docker-server> --docker-username=<docker-username> --docker-password=<docker-password> 
+    ```
+
+2. add this pull secret to the ibm-infra-management-application-default service account, for example:
+
+    ```
+    oc secret link ibm-infra-management-application-default <pull-secret-name> --for=pull
+    ```
```

Comment:
This should also use conrefs, and also be sentence cased.

```suggestion
2. Add this pull secret to the `{{ site.data.product.workflow_service_account }}` service account, for example:

    ```
    oc secret link {{ site.data.product.workflow_service_account }} <pull-secret-name> --for=pull
    ```
```